### PR TITLE
Feature/helper functions

### PR DESCRIPTION
**Pull request by SeanPeterson**
## Info
Twig functions for common bad elements used throughout modules.

### CODE REVIEW

File: src/Twig/BadgeExtension.php
Comment (justin-oh) [inline src/Twig/BadgeExtension.php:1]: We'll need a function for all 8 variations: https://getbootstrap.com/docs/5.3/components/badge/#background-colors

**Review by justin-oh — CHANGES_REQUESTED** [review]
Note about adding more badge functions.

Also, everything for Symfony needs to be ran through php-cs-fixer. Noticed some deviations with the string concatenation.

**Review by justin-oh — COMMENTED** [review]
Sorry, I'm thinking this should go into the bootstrap bundle, in which case it should be prefixed like everything else in that bundle. We're implementing it as is in [the documentation](https://getbootstrap.com/docs/5.3/components/badge/#background-colors), if we ignore the pill effect. (I don't foresee a reason to support both badge styles.)

I think I will eventually update the prefix from `bootstrap_` to `bs_` to shorten it. These functions are used across several bundles so it's not a simple update.

**Comment by SeanPeterson** [comment]
> the documentation

Now that you say that, I can't get string concatination auto-formatting to work. I think it should turn `>'.$message.'</span>'` into `>' . $message . '</span>'`?



I have the executable set to `~/ohdev/tools/php-cs-fixer/vendor/bin/php-cs-fixer`
Config set to `.php-cs-fixer.php;.php-cs-fixer.dist.php;.php_cs;.php_cs.dist`
and `  "php-cs-fixer.rules": "@PSR12",`

**Comment by SeanPeterson** [comment]
fat finger close it - though it sounds like this may go into the boostrap bundle anyways

**Comment by justin-oh** [comment]
@SeanPeterson Concatenation formatting removes space before and after the `.`.

The rules should be `@Symfony` instead of `@PSR12`.

**Pull request closed without merging**